app/src/main/java/com/kks/nimblesurveyjetpackcompose/di/ViewModelModule.kt

```diff
@@ -0,0 +1,23 @@
+package com.kks.nimblesurveyjetpackcompose.di
+
+import com.kks.nimblesurveyjetpackcompose.model.ApiInterface
+import com.kks.nimblesurveyjetpackcompose.repo.login.LoginRepo
+import com.kks.nimblesurveyjetpackcompose.repo.login.LoginRepoImpl
+import com.kks.nimblesurveyjetpackcompose.util.CustomKeyProvider
+import com.kks.nimblesurveyjetpackcompose.util.PreferenceManager
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ViewModelComponent
+
+@Module
+@InstallIn(ViewModelComponent::class)
+object ViewModelModule {
+
+    @Provides
+    fun provideLoginRepoImpl(
```

Comment:
The naming is not too important for the provider methods, but IMO, `provideLoginRepo` makes more sense.
```suggestion
    fun provideLoginRepo(
```